browser/components/molecules/PendingPrrrs/index.js

```diff
@@ -28,9 +28,16 @@ export default class PendingPrrrs extends Component {
         console.warn('Claim Error')
         console.error(error)
         this.setState({error})
+        this.startErrorTimeout()
       })
   }
 
+  startErrorTimeout() {
+    window.setTimeout(() => {
+      this.setState({error: null})
+    }, 5000)
+  }
```

Comment:
this is a good start. But we want to make sure we dont remove the error sooner than 5 seconds. Imagine the scenario:
0. user doest something that causes an error
0. we show the error and schedule the error removal for 5 seconds from now
0. 3 seconds go by
0. user doest something that causes another error
0. the error will be removed too quickly. 

To solve this lets make sure that when an error occurs we use `clearTimeout` to disable any previously scheduled error removal timeoutes.